node/chainSimulator/components/cryptoComponents.go

```diff
@@ -60,10 +59,8 @@ func CreateCryptoComponents(args ArgsCryptoComponentsHolder) (factory.CryptoComp
 		ActivateBLSPubKeyMessageVerification: true,
 		IsInImportMode:                       false,
 		ImportModeNoSigCheck:                 false,
-		P2pKeyPemFileName:                    "",
-		ValidatorKeyPemFileName:              args.ValidatorKeyPemFileName,
-		AllValidatorKeysPemFileName:          "",
-		SkIndex:                              args.SkIndex,
+		ValidatorKeyPemFileName:              "missing.pem",
```

Comment:
a constant here? or an additional comment for why using this file?

Reply:
added a comment